src/chat/ChatScreen.tsx

```diff
@@ -161,8 +161,8 @@ export const ChatScreen: React.FC<ChatScreenProps> = ({
         (message: MessageProps) => {
           if (userInfo && message.senderId !== userInfo.id) {
             const userInfoIncomming = {
-              id: message.id,
-              name: message.senderId,
+              id: message.senderId,
+              name: partners?.find((e) => e.id === message.senderId)?.name,
```

Comment:
I think "partner" is not the right word for this situation. How about "participants"?